espnet/nets/pytorch_backend/tacotron2/encoder.py

```diff
@@ -162,3 +162,127 @@ def inference(self, x):
         ilens = [x.size(0)]
 
         return self.forward(xs, ilens)[0][0]
+
+
+class EncoderNoEmb(torch.nn.Module):
```

Comment:
If the difference is only the input layer, I think it is not necessary to separate the module.